packaging/installer/netdata-updater.sh

```diff
@@ -263,6 +265,43 @@ get_latest_version() {
   fi
 }
 
+update_available() {
+  basepath="$(basename "$(basename "$(basename "${NETDATA_LIB_DIR}")")")"
+  searchpath="${basepath}/bin:${basepath}/sbin:${basepath}/usr/bin:${basepath}/usr/sbin:${PATH}"
+  searchpath="${basepath}/netdata/bin:${basepath}/netdata/sbin:${basepath}/netdata/usr/bin:${basepath}/netdata/usr/sbin:${searchpath}"
+  ndbinary="$(env PATH="${searchpath}" command -v netdata 2>/dev/null)"
+
+  if [ -z "${ndbinary}" ]; then
+    current_version=0
+  else
+    current_version="$(parse_version "$(${ndbinary} -v | cut -f 2 -d ' ')")"
+  fi
+
+  latest_tag="$(get_latest_version)"
+  latest_version="$(parse_version "${latest_tag}")"
+  path_version="$(echo "${latest_tag}" | cut -f 1 -d "-")"
+
+  # If we can't get the current version for some reason assume `0`
+  current_version="${current_version:-0}"
+
+  # If we can't get the latest version for some reason assume `0`
+  latest_version="${latest_version:-0}"
+
+  info "Current Version: ${current_version}"
+  info "Latest Version: ${latest_version}"
+
+  if [ "${latest_version}" -gt 0 ] && [ "${current_version}" -gt 0 ] && [ "${current_version}" -ge "${latest_version}" ]; then
+    info "Newest version (current=${current_version} >= latest=${latest_version}) is already installed"
+    return 1
+  elif [ -n "${NETDATA_TARBALL_CHECKSUM}" ] && grep "${NETDATA_TARBALL_CHECKSUM}" sha256sum.txt >&3 2>&3; then
```

Comment:
This doesn't work because `sha256sum.txt` files is not downloaded at this moment.